app/src/components/main/modal/modalLayout/modalFooter/modalFooter.scss

```diff
@@ -55,6 +61,10 @@
     font-size: 10px;
     line-height: 1.4;
     color: $COLOR--failed;
+
+    &.color-orange {
+      color: $COLOR--black-1;
+    }
```

Comment:
This confuses. color orange but actually black
Maybe better provide warning type or appearance and rely on it f.e we have warningType = warn, so he is responsible for making the icon orange and the text black
error make them both red
and so on

Reply:
Yeah, I agree, but in this case we have to update all used footers

Reply:
I think its Ok to update all warning appearance as they are rendered by same modal layout.